plugins/3rd-party-optimizer/package.json

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "3rd-party-optimizer",
```

Comment:
100% subjective and a huge nit: `third-party-optimizer`? 😁

Reply:
I like `3rd` more. You'll usually see `3rd-party scripts` or `3rd-party trackers` or even `3rd-party cookies`, not `Third` in any of the terms